settings.py

```diff
@@ -2065,6 +2065,8 @@ def _determine_couch_databases(couch_databases):
     ('enikshay_date_of_acceptance', 'custom.enikshay.expressions.date_of_acceptance_expression'),
     ('enikshay_episode_from_person', 'custom.enikshay.expressions.episode_from_person_expression'),
     ('enikshay_key_populations', 'custom.enikshay.expressions.key_populations_expression'),
+    ('enikshay_key_populations', 'custom.enikshay.expressions.most_recent_referral_expression'),
```

Comment:
i think you forgot to change the first part of the tuple when you copied and pasted for both of the new expressions